src/menu/views/fault.c

```diff
@@ -10,7 +10,7 @@ static void draw (menu_t *menu, surface_t *d) {
         "Supported firmware versions:\n"
         "64drive: 2.05+\n"
         "EverDrive-64: ???+\n"
-        "SummerCart64: 2.17.0+"
+        "SummerCart64: 2.20.0+"
```

Comment:
this is misleading because the actual version check is defined here https://github.com/Polprzewodnikowy/N64FlashcartMenu/blob/develop/src/flashcart/sc64/sc64.c#L23-L25

Reply:
also there are no new firmware features that menu could use so bumping the required firmware version will only be annoying to the end user

Reply:
That depends on how you look at it... there is "supported" (by the developer and docs) and "supported" by the flashcart and menu... but this error is not "minimum supported", so I was going for "what you should be using"...

Reply:
If we are going for min versions, then perhaps we should use the defs?!

Reply:
this error screen will be triggered _only_ on firmwares below 2.17.0, and basically no one use that old version
you can change this message to use the version defines for SC64 and 64drive

Reply:
Not trying to dodge this, but it became a burden, so I took the easy path.